### DISCUSSION TITLE
Issue with password validation

### DISCUSSION THREAD

**Discussion by CovertError** [livewire/livewire#5045] [1. Help]
how does this work 
<img width="601" alt="image" src="https://user-images.githubusercontent.com/34497804/174428074-792d6372-dccb-4282-81d5-743bec1abed7.png">
and this doesnt 
<img width="601" alt="image" src="https://user-images.githubusercontent.com/34497804/174428088-fc091b02-2f8b-4de5-976c-df76ace60c51.png">

**Comment by michael-rubel** [comment]
Hey @CovertError 

You cannot use an expression in default property values.
To apply the custom rules, you can define `public function rules()` in your component instead of the class property.

You can even get the rules from Form Request, like in this example:
https://github.com/michael-rubel/livewire-best-practices/blob/main/Examples/form-request.md
